### DISCUSSION TITLE
PPA with 18.04 Bionic

### DISCUSSION THREAD

**Discussion by stephematician** [abraunegg/onedrive#1374] [Q&A] [answered]
I tried adding the ppa to /etc/apt/sources.list in Ubuntu 18.04

```bash
sudo add-apt-repository ppa:yann1ck/onedrive
...
Err:22 https://ppa.launchpad.net/yann1ck/onedrive/ubuntu bionic Release                   
  404  Not Found [IP: 91.189.95.85 80]
....
E: The repository 'http://ppa.launchpad.net/yann1ck/onedrive/ubuntu bionic Release' does not have a Release file.
```

Having a look at the [PPA](https://launchpad.net/~yann1ck/+archive/ubuntu/onedrive) it looks like 18.04 is not included.

Any chance a package can be created - otherwise I'll have to go the build myself route.

**Comment by abraunegg** [comment, marked as answer]
@stephematician 
Correct - the PPA does not support Ubuntu 18.x

The reasons for this are:
* DMD compiler is not available
* LDC compiler version is less than the minimum required to compile the client

For Ubuntu 18.x your only option is to compile from source.

There are 2 alternatives to compiling:
* Upgrade to Ubuntu 20.x
* Use a different Linux distribution that is not Ubuntu based, where the client is pre-packaged for that distribution

**Reply by stephematician** [reply]
Thanks. I had no idea about the discussion section - so this is an accidental duplicate of #1369. Apologies.
